Sources/Latte+Flows.swift

```diff
@@ -130,6 +130,62 @@ public extension Latte {
         }
     }
 
+    func changePassword(
+        context: UINavigationController,
+        handler: @escaping ResultHandler<UserInfo>
+    ) {
+        Task { await run() }
+
+        @Sendable @MainActor
+        func run() async {
+            var viewController: LatteViewController?
+            do {
+                let entryURL = customUIEndpoint + "/settings/change_password"
+                let redirectURI = "latte://complete"
+
+                let urlQueryAllowed = CharacterSet.urlQueryAllowed.subtracting(["+"])
+                let query = [
+                    "redirect_uri=\(redirectURI.addingPercentEncoding(withAllowedCharacters: urlQueryAllowed)!)"
+                ].joined(separator: "&")
+                let entryURLWithWQuery = "\(entryURL)?\(query)"
+
+                let url: URL = try await withCheckedThrowingContinuation { resume in
+                    authgear.experimental.generateURL(redirectURI: entryURLWithWQuery) {
+                        resume.resume(with: $0)
+                    }
+                }
+
+                let webViewRequest = LatteWebViewRequest(url: url, redirectURI: redirectURI)
+                let latteVC = LatteViewController(context: context, request: webViewRequest)
+                viewController = latteVC
+
+                let result: LatteWebViewResult = try await withCheckedThrowingContinuation { next in
+                    latteVC.handler = { next.resume(with: $0) }
+                    context.pushViewController(latteVC, animated: true)
+                }
+
+                let components = URLComponents(url: result.finishURL, resolvingAgainstBaseURL: false)!
+                if let urlError = components.queryItems?.first(where: { $0.name == "error" })?.value {
+                    let error: AuthgearError
+                    if urlError == "cancel" {
+                        error = .cancel
+                    } else {
+                        error = .oauthError(OAuthError(error: urlError, errorDescription: nil, errorUri: nil))
+                    }
+                    handler(Handle(isPresented: false, viewController: viewController, result: .failure(error)))
+                    return
+                }
+
+                let userInfo: UserInfo = try await withCheckedThrowingContinuation { resume in
+                    authgear.experimental.authgear.fetchUserInfo() { resume.resume(with: $0) }
```

Comment:
Not sure if we need fetch userinfo; unlike verify email, changing password would not change the userinfo.